admin/install_web_server.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/sh
 
-apt-get -y install memcached node-less python-virtualenv python-dev ipython
+apt-get -y install memcached node-less python3.4-venv python3-dev ipython3
```

Comment:
What python are we targeting in other MeB projects? It might be worth making this one the same. 3.5, 3.6?